package.json

```diff
@@ -269,6 +270,7 @@
         "@hapi/address": "^2.0.0",
         "axios": "^0.18.1",
         "classnames": "^2.2.5",
+        "color": "^3.1.2",
```

Comment:
should add breaking change to PR

Reply:
👍 do we do that because of the change to a peerdep? or is that because of the minor version bump from 3.0.x -> 3.1.x